src/patternfly/components/ChipGroup/chip-group.scss

```diff
@@ -3,10 +3,10 @@
 .pf-c-chip-group {
   @include pf-t-light; // force the container follow the light theme
 
-  --pf-c-chip-group--MarginRight: calc(var(--pf-c-chip-group--c-chip--MarginRight) * -1);
+  --pf-c-chip-group--MarginRight: var(--pf-global--spacer--xs);
```

Comment:
Do we need this margin? It's creating this extra space on the right here

<img width="108" alt="Screen Shot 2019-10-23 at 12 48 18 PM" src="https://user-images.githubusercontent.com/35148959/67420016-6ed59700-f593-11e9-909d-2c6807799e21.png">

If it's to space the group from the action, could we apply a left margin to the action?